libs/stats/api/src/lib/stats.resolver.ts

```diff
@@ -0,0 +1,23 @@
+import {Query, Resolver} from '@nestjs/graphql'
+import {StatsService} from './stats.service'
+import {GraphQLISODateTime} from '@nestjs/graphql'
+
+@Resolver()
+export class StatsResolver {
+  constructor(private statsService: StatsService) {}
+
+  @Query(() => Number)
```

Comment:
Should probably be using `ResolveField` here and not `Query`, else you need to fire multiple queries. `@Resolver` + `@ResolveField` allows you to lazyload fields.